internal/characters/thoma/thoma.go

```diff
@@ -42,6 +42,17 @@ func (c *char) Init() error {
 		c.c6buff = make([]float64, attributes.EndStatType)
 		c.c6buff[attributes.DmgP] = .15
 	}
+
+	common.NewNAHook(
```

Comment:
Please consider passing a `common.NAHook` struct directly to make it easier to read/harder to mix up the params.